src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1435,8 +1435,12 @@ public partial class StringReader : System.IO.TextReader
         public override int Peek() { throw null; }
         public override int Read() { throw null; }
         public override int Read(char[] buffer, int index, int count) { throw null; }
+        public override int Read(System.Span<char> destination) { throw null; }
```

Comment:
nit: Is specifying System required in the type name? If not, remove (here and elsewhere) to be consistent?

Reply:
Using the full type name is consistent with how the ref generator tool generates the code.